<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EPBYMINW1906" name="ReaderTest" tests="1" failures="1" timestamp="27 Apr 2015 09:34:45 GMT" time="0.004" errors="0">
  <testcase name="readAllTest" time="0.004" classname="ReaderTest">
    <failure type="java.lang.AssertionError" message="readAllTest expected:&lt;Id: 3, Type: Cruise, Food: Breakfast, Transport: Azura, Duration: 30, Price: 550000.0; Countries: [Hawaii,Caribbean,Bahamas]&gt; but was:&lt;Id: 3, Type: Cruise, Food: Breakfast, Transport: Azura, Duration: 30, Price: 550000.0; Countries: [Hawaii, Caribbean, Bahamas]&gt;">
      <![CDATA[java.lang.AssertionError: readAllTest expected:<Id: 3, Type: Cruise, Food: Breakfast, Transport: Azura, Duration: 30, Price: 550000.0; Countries: [Hawaii,Caribbean,Bahamas]> but was:<Id: 3, Type: Cruise, Food: Breakfast, Transport: Azura, Duration: 30, Price: 550000.0; Countries: [Hawaii, Caribbean, Bahamas]>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:171)
	at ReaderTest.readAllTest(ReaderTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at TestRunner.main(TestRunner.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
]]>
    </failure>
  </testcase>
</testsuite>
